### ISSUE TITLE
Misleading error message when running on non-compatible php version

### ISSUE THREAD

**Issue by krchiper** [maglnet/ComposerRequireChecker#465]
When running `composer-require-checker` on older php version you get a message indicating that you dependencies requires php 8.2, but its really `composer-require-checker` that requires php 8.2.

```
composer-require-checker.phar
Composer detected issues in your platform:

Your Composer dependencies require a PHP version ">= 8.2.0". You are running 8.1.26.

PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.2.0". You are running 8.1.26. in phar:///usr/local/bin/composer-require-checker.phar/vendor/composer/platform_check.php on line 24
```

**Comment by Ocramius** [comment]
This is correct: that's just how the Composer autoloader is baked in `.phar` files though, so I suggest you raise the issue for broader discussion in Phing.

See https://github.com/maglnet/ComposerRequireChecker/blob/f49bc1e16a566a02c768fc4b71419eb0bbd0e4ef/.github/workflows/phar-creation.yml#L31-L35
